### ISSUE TITLE
sakia 0.30.13 crashes

### ISSUE THREAD

**Issue by galuel** [duniter/sakia#635]
Launching sakia 0.30.13 on Ubuntu 16.10 / 64 bits : 
```bash
(sakia:17139): Gtk-WARNING **: Theme parsing error: calendar.css:9:18: 'outline-radius' is not a valid property name
ERROR:root:An unhandled exception occured
Traceback (most recent call last):
  File "sakia/main.py", line 96, in <module>
  File "sakia/app.py", line 87, in startup
  File "sakia/app.py", line 100, in load_profile
  File "sakia/data/repositories/meta.py", line 44, in load_or_init
  File "sakia/data/repositories/meta.py", line 77, in upgrade_database
  File "sakia/data/repositories/meta.py", line 98, in add_ud_rythm_parameters
FileNotFoundError: [Errno 2] Aucun fichier ou dossier de ce type: '/opt/sakia/sakia/data/repositories/000_add_ud_rythm_parameters.sql'
Failed to execute script main
LOADER: OK.
LOADER: Cleaning up Python interpreter.
ERROR:root:Task was destroyed but it is pending!
task: <Task pending coro=<get_last_version() running at sakia/app.py:188>>
sys:1: RuntimeWarning: coroutine 'get_last_version' was never awaited
LOADER: Back to parent (RC: 255)
LOADER: Doing cleanup
LOADER: Freeing archive status for /opt/sakia/sakia.bin
```

**Comment by galuel** [comment]
Back to Sakia 0.30.11 old version works... New release expected :)

**Comment by Insoleet** [comment]
Oups :) Fixed in 0.30.14.
